libflux/flux-core/src/scanner/scanner.rl

```diff
@@ -23,6 +23,7 @@ use crate::scanner::*;
         // so we can record the location of the first byte of a token.
         last_newline_before_token = *last_newline;
         cur_line_token_start = *cur_line;
+        cur_line_token_start = *cur_line;
```

Comment:
it looks like just a whitespace change

Reply:
i think the line has been duplicated see https://github.com/influxdata/flux/blob/d387dc4e2866ee90917af934817f8a3fe1cb7678/libflux/flux-core/src/scanner/scanner.rl#L26 can you remove the duplicate?

Reply:
Oops. Will do when I get back at a keyboard tomorrow.

Reply:
fixed